vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "redirects": [
+    {
+      "source": "/discord",
+      "destination": "https://discord.com/invite/WyPN5r9",
+      "permanent": false
+    }
+  ]
+}
```

Comment:
Nice!

Just doublechecking: Is this automatically deployed every time?

Reply:
I think so, I tried on https://website-6tj5dd38a-game-ci.vercel.app/discord and it does the redirect properly 👍

Reply:
I checked it too, so you didn't use the vercel CLI manually at all?